src/main/scala/org/globalforestwatch/summarystats/carbon_sensitivity/CarbonSensitivitySummary.scala

```diff
@@ -23,15 +23,18 @@ case class CarbonSensitivitySummary(
 object CarbonSensitivitySummary {
   // CarbonSensitivitySummary form Raster[CarbonSensitivityTile] -- cell types may not be the same
 
-  implicit val mdhCellRegisterForCarbonSensitivityRaster1: CellVisitor[Raster[CarbonSensitivityTile], CarbonSensitivitySummary] =
-    new CellVisitor[Raster[CarbonSensitivityTile], CarbonSensitivitySummary] {
+  implicit val mdhCellRegisterForTreeLossRaster1
+  : GridVisitor[Raster[CarbonSensitivityTile], CarbonSensitivitySummary] =
+    new GridVisitor[Raster[CarbonSensitivityTile], CarbonSensitivitySummary] {
+      val acc = new CarbonSensitivitySummary()
 
-      def register(
+      def result = acc
+
+      def visit(
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)